scuba/config.py

```diff
@@ -155,9 +155,12 @@ def override(self, node: yaml.nodes.Node) -> OverrideMixin:
         assert isinstance(obj, OverrideMixin)
         return obj
 
+    def not_yet_implemented(self, node: yaml.nodes.Node):
+        pass
 
 Loader.add_constructor("!from_yaml", Loader.from_yaml)
 Loader.add_constructor("!override", Loader.override)
+Loader.add_constructor("!reference", Loader.not_yet_implemented)
```

Comment:
Can you update your PR description to reference #200 which spells out in great detail what the `!reference` tag is, and how GitLab uses it?